### PR TITLE
Add IP entity

### PR DESCRIPTION
**Pull request by thomasgermain**
see #104

I tested with shelly 1, shelly 2.5 and shelly dimmer 2.

I added the IP entity everywhere the `rssi` is.

### CODE REVIEW

File: python_scripts/shellies_discovery.py [hunk 1]
```diff
@@ -662,16 +665,17 @@ def mqtt_publish(topic, payload, retain, qos):
...
         DEVICE_CLASS_SIGNAL_STRENGTH,
         None,
         DEVICE_CLASS_TIMESTAMP,
+        None
     ]
-    sensors_units = [UNIT_CELSIUS, UNIT_DB, None, None]
-    sensors_tpls = [TPL_TEMPERATURE, TPL_RSSI, TPL_SSID, TPL_UPTIME]
-    sensors_topics = [None, None, None, None]
+    sensors_units = [UNIT_CELSIUS, UNIT_DB, None, None, None]
+    sensors_tpls = [TPL_TEMPERATURE, TPL_RSSI, TPL_SSID, TPL_UPTIME, None]
```
Comment (bieniu) [inline python_scripts/shellies_discovery.py:677]:
```suggestion
    sensors_tpls = [TPL_TEMPERATURE, TPL_RSSI, TPL_SSID, TPL_UPTIME, TPL_IP]
```

File: python_scripts/shellies_discovery.py [hunk 2]
```diff
@@ -1088,25 +1099,26 @@ def mqtt_publish(topic, payload, retain, qos):
...
     bin_sensors_topics = [TOPIC_INPUT_0, TOPIC_LONGPUSH_0, TOPIC_LONGPUSH_0, TOPIC_INFO]
-    sensors = [SENSOR_RSSI, SENSOR_SSID, SENSOR_UPTIME]
-    sensors_units = [UNIT_DB, None, None]
-    sensors_classes = [DEVICE_CLASS_SIGNAL_STRENGTH, None, DEVICE_CLASS_TIMESTAMP]
-    sensors_tpls = [TPL_RSSI, TPL_SSID, TPL_UPTIME]
-    sensors_topics = [None, None, None]
+    sensors = [SENSOR_RSSI, SENSOR_SSID, SENSOR_UPTIME, SENSOR_IP]
+    sensors_units = [UNIT_DB, None, None, None]
+    sensors_classes = [DEVICE_CLASS_SIGNAL_STRENGTH, None, DEVICE_CLASS_TIMESTAMP, None]
+    sensors_tpls = [TPL_RSSI, TPL_SSID, TPL_UPTIME, None]
```
Comment (bieniu) [inline python_scripts/shellies_discovery.py:1105]:
```suggestion
    sensors_tpls = [TPL_RSSI, TPL_SSID, TPL_UPTIME, TPL_IP]
```

**Review by bieniu — CHANGES_REQUESTED** [review]

**Comment by bieniu** [comment]
Please format the code with `Black`.

**Comment by thomasgermain** [comment]
it should be fine now 😃

**Review by bieniu — APPROVED** [review]

**Comment by bieniu** [comment]
Good job, thanks 👍

**Pull request merged**